### ISSUE TITLE
Nagrand outpost mount prevents while-moving speed boost

### ISSUE THREAD

**Issue by ckaotik** [Phanx/MountMe#5]
The addon prioritizes the mounts gained from the nagrand garrison outpost over regular mounts, which is good. However, at the same time the class specific speed boosts that are usable while moving get disabled, e.g. ghost wolf while moving.

**Comment by Phanx** [comment]
Yes, the current handling of the Nagrand mounts isn't ideal, but they don't count as "mounted" according to the API, so there's no good solution. There's a buff you can check for, but checking for buffs isn't allowed in the secure environment (eg. macro conditionals) so while it would be possible to work better out of combat, there's no in-combat improvement possible.

**Comment by ckaotik** [comment]
What if you only considered the Nagrad mount when not moving? I'm not that good with the secure environment, but considering you already can differentiate between "moving", "combat" and "all other", it might be doable?

- player is moving - use moving spell.
- player is in nagrand - use nagrand spell. (applies to both combat and nocombat)
- player is in combat - use combat spell.
- none of the above - use random mount.

**Comment by Phanx** [comment]
It's only possible to check "moving" out of combat, but that's okay because the only action you can't do while moving (summoning a mount) isn't possible in combat either.

The problem with the Nagrand mount is that it's usable in combat, but you can't check for it in combat.

There's definitely room for improvement out of combat, but given how little time I spend playing, and how little of that time I spend in Nagrand, it's kind of at the bottom of the priority list.
